.github/workflows/check_vcpkg.yaml

```diff
@@ -0,0 +1,86 @@
+name: vcpkg Update
+concurrency:
+  group: vcpkg-${{ github.ref }}
+  cancel-in-progress: true
+on:
+  workflow_dispatch:
+  schedule:
+    # Weekly on Friday at 2:00 a.m.
+    - cron: '0 2 * * 5'
```

Comment:
This clashes with the translations update if it happens on Friday. It looks like translations usually take about 16 minutes, but there was one 3 weeks ago that went over 2 hours. vcpkg deps need about 1 hour.

I think the trivial solution would be to schedule translations update on fixed weekdays instead of every 3rd day.